pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -585,6 +585,30 @@ class Build {
         return remoteTargets
     }
 
+   def compareReproducibleBuild() {
+        //Currently only enable for jdk17+, linux_x64, temurin, nightly, which shouldn't affect current build
+        def jobName = "${env.JOB_NAME}_reproduce_compare"
+        if (getJavaVersionNumber() >= 17 &&
```

Comment:
At the moment it looks like we only have the jobs for 17 - should this be changed to equals instead of >= ?

Reply:
Yes, updated.